googlemock/CMakeLists.txt

```diff
@@ -135,11 +131,7 @@ if (gmock_build_tests)
   enable_testing()
 
   if (MINGW OR CYGWIN)
-    if (CMAKE_VERSION VERSION_LESS "2.8.12")
-      add_compile_options("-Wa,-mbig-obj")
-    else()
-      add_definitions("-Wa,-mbig-obj")
```

Comment:
`add_definitions` is used to add defines to the code like `NOMINMAX` or `WIN32_LEAN_AND_MEAN`  using it for compiler options isn't correct.

Also `add_definitions` isn't recommended for usage anymore:
https://cmake.org/cmake/help/latest/command/add_definitions.html#command:add_definitions